include/swift/Reflection/ReflectionContext.h

```diff
@@ -101,6 +101,9 @@ class ReflectionContext
   std::vector<MemoryReader::ReadBytesResult> savedBuffers;
   std::vector<std::tuple<RemoteAddress, RemoteAddress>> imageRanges;
 
+  bool supportsPriorityEscalation = false;
```

Comment:
@mikeash, our changes to RuntimeInternals.h caused a bunch of unhappiness here and so I took a stab at fixing it, including stealing some logic from your PR #41160.

Would appreciate a look-over.

Reply:
Looks good! We'll get some conflicts and need some more fixes once one of us merges, but this is good for the moment.